docs/en/reference/information_schema/materialized_views.md

```diff
@@ -0,0 +1,34 @@
+---
+displayed_sidebar: "English"
+---
+
+# materialized_views
+
+`materialized_views` provides information about all asynchronous materialized views.
+
+The following fields are provided in `materialized_views`:
+
+| **Field**                            | **Description**                                              |
+| ------------------------------------ | ------------------------------------------------------------ |
+| MATERIALIZED_VIEW_ID                 | ID of the materialized view.                                 |
+| TABLE_SCHEMA                         | Database in which the materialized view resides.             |
+| TABLE_NAME                           | Name of the materialized view.                               |
+| REFRESH_TYPE                         | Refresh type of the materialized view, including `ROLLUP`, `ASYNC`, and `MANUAL`. |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/b9aa1060-3415-4556-b9ea-d6aa502c90f1)
-> Valid values: `ROLLUP`, `ASYNC`, and `MANUAL`.

Reply:
`Valid values` are more like to describe configurable parameters rather than returned values?